### PR TITLE
🚩: Milestone 2: Components | component- >>branch

### PR DESCRIPTION
**Pull request by Ruthmy**
### In this new Branch, We have **ADDED** the succeeding **\*📂 folders** and **\*\*📄 files**.

- ### 📂\* root/src/
   - ### 📂\* components/

      - \*\*📄 Calculator.js

          > As the main Component that contain the `Display` and `ButtonPanel`.

      - \*\*📄 Display.js

          > Component that serves as a display

      - \*\*📄 Button.js

          > Base component for the creation of buttons

      - \*\*📄 ButtonPanel.js

          > Component that renders the calculator buttons from an array.

 
      - \*\*📄 Display.js

      - \*\*📄 ButtonPanel.js

          > Add the base styles for the calculator components.

### In this Branch, We have **MODIFIED** the succeeding **📂\* folders** and **📄\*\* files**

- ### 📂\* root/src/
    - \*\*📄 index.js

         > Deleted the reportWebVitals 

    - \*\*📄 App.js

         > Deleted the logo and the React app boiler plate.

    - \*\*📄 index.css

    - \*\*📄 App.css

         > Modified the default styles and added new class selectors and style properties to style the calculator.

    - \*\*📄 README.md

         > Updated the Future Requirements.

### In this Branch, We have **DELETED** the succeeding **📂\* folders** and **📄\*\* files**

- ### 📂\* root/src/
     - \*\*📄 App.test.js
     - \*\*📄 Applogo.svg
     - \*\*📄 AppreportWebVitals.js
     - \*\*📄 AppsetupTests.js

---

### Thank you for taking the time to review this PR ⭐!

---

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+// import { Button } from './Button';
```
Comment (Munsa1) [inline src/components/Calculator.js]: - [x] Kindly remove commented code for a cleaner codebase and more readable code

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+const ButtonPanel = () => (
+  <div className="ButtonPanel">
+    {buttonList.map((button) => (
+      <Button key={button.label} value={button.label} className={button.class}>
+        {button.label}
+      </Button>
+    ))}
+  </div>
+);
```
Comment (Munsa1) [inline src/components/ButtonPanel.js:34]: - [x] Kindly follow the given design for the calculator as this is a core requirement.

Your design
image

Given design
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Ruthmy, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Created Atleast One Child Component :heavy_check_mark: 
Removed All Boilerplate Code :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (Ruthmy) [reply src/components/Calculator.js]: Hi @Munsa1 !
Thanks for your feedback, you're right. I totally forgot to delete that comment.

File: src/components/ButtonPanel.js (hunk 2 above)
Comment (Ruthmy) [reply src/components/ButtonPanel.js:34]: Hello @Munsa1 !
Thanks for your feedback. 
In the requirements there is no design to follow, there is a medium-fidelity wireframes, but I recognize that I took several liberties and with them came an overflow, so I modified the base structure to correct it.
Thank you very much again.

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @Ruthmy  ,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉



Highlights
- [ ] Good code organization ✔️
- [ ] Descriptive PR✔️

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**